common/src/main/java/com/redhat/cloud/notifications/routers/dailydigest/TriggerDailyDigestRequestDto.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.cloud.notifications.routers.dailydigest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import javax.validation.constraints.NotBlank;
+import java.beans.ConstructorProperties;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+import java.util.Objects;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class TriggerDailyDigestRequestDto {
+
+    @JsonProperty("application_name")
+    @NotBlank
+    private final String applicationName;
```

Comment:
I guess you also need to mark them as `@Required`  - i.e. they can be null but not blank (IIRC)

Reply:
This is not needed! The `@NotBlank` annotation makes sure that the target `String` isn't null, empty or blank!